src/main/java/com/google/sps/data/DataGraph.java

```diff
@@ -0,0 +1,107 @@
+package com.google.sps.data;
```

Comment:
Skipping this file because I think it is part of PR#20? I can re-review once that PR is merged.